apps/api/v2/src/app.ts

```diff
@@ -41,16 +41,17 @@ export const bootstrap = (app: NestExpressApplication): NestExpressApplication =
     })
   );
 
+  const { httpAdapter } = app.get(HttpAdapterHost);
+  app.useGlobalFilters(new SentryFilter(httpAdapter));
```

Comment:
according to nest.js doc: `When combining an exception filter that catches everything with a filter that is bound to a specific type, the "Catch anything" filter should be declared first to allow the specific filter to correctly handle the bound type.`

Reply:
I think that this deserves abstracting filter setup into functions, so that in the future developers do not add specific filters before the catch all Sentry one:

setupCatchAllExceptionFilter(app);
setupSpecificExceptionFilters(app);
setupHttpExceptionFilter(app);